pyam/core.py

```diff
@@ -1284,6 +1287,48 @@ def export_meta(self, excel_writer, sheet_name='meta'):
         if close:
             excel_writer.close()
 
+    def to_datapackage(self, path, tmp='tmp'):
+        """Write object to a frictionless datapackage
+
+        More information: https://frictionlessdata.io
+
+        Returns the saved :class:`datapackage.Package`. When adding metadata
+        (descriptors), please follow the `template` defined by
+        https://github.com/OpenEnergyPlatform/metadata
+
+        Parameters
+        ----------
+        path: string
```

Comment:
```suggestion
        path: string or pathlib.Path
```

a `Path` of a `Path` remains the same `Path` so no changes required below